projects/components/src/popover/popover.component.ts

```diff
@@ -90,4 +121,29 @@ export class PopoverComponent implements OnDestroy {
 
     this.popoverOpen.emit(this.popover);
   }
+
+  private handleHoverPopoverTrigger(event: 'mouseenter' | 'mouseleave'): void {
+    if (this.disabled) {
+      return;
+    }
+
+    switch (event) {
+      case 'mouseenter':
+        this.popoverHoverTriggerService.showPopover({
+          origin: this.popoverElement,
+          popoverContent: this.content,
+          options: {
+            data: this.content.data
+          }
+        });
+        break;
+
+      case 'mouseleave':
+        this.popoverHoverTriggerService.closePopover();
+        break;
+
+      default:
+        assertUnreachable(event);
```

Comment:
typically want to have switch cases return and return this one too. It's moot given the comment about getting rid of the switch though.